.github/workflows/nextjs.yml

```diff
@@ -37,18 +34,15 @@ jobs:
       - name: Get current date
         id: date
         run: echo "DATE=$(date +'%Y-%m')" >> $GITHUB_ENV
-      - name: Restore conda downloads cache
-        uses: actions/cache@v4
-        with:
-          path: |
-            ~/.dashboard
-          key: conda-download-cache-${{ env.DATE }}
```

Comment:
For the benefit of future readers, we discussed this offline: this chunk was missed in #27, it's technically orthogonal to this PR.